### DISCUSSION TITLE
Replace default leader-w mappings

### DISCUSSION THREAD

**Discussion by blancpain** [LazyVim/LazyVim#4085] [Q&A] [answered]
Hi all,

Is there an easy/reliable way to disable <leader-w> opening the window management and keep it to only <c-w>? I am using <leader-w> to save buffer but recent changes to which-key/lazyvim seem to have broken my config and can't figure out how to disable the default <leader-w> mapping...

Thanks!

**Comment by dpetka2001** [comment, marked as answer]
Do you mean `<leader>w` to save to buffer and you don't want the default `<leader>w` which-key mapping? If so, you could try setting 
```lua
{
  "folke/which-key.nvim",
  opts = {
    spec = {
      { "<leader>w", hidden = true },
    },
  },
},
```
and make a keymap for `<leader>w` to write to buffer.

**Reply by blancpain** [reply]
Awesome - this worked indeed! Is there a way to now add a description to my new leader-w mapping (e.g. "Save buffer") that appears in the which-key window? 

I have the below in my keymaps file and it works but it doesn't add the desc: 

```lua
vim.keymap.set({ "x", "n", "s" }, "<leader>w", "<cmd>w<cr><esc>", { desc = "Save File" })
```

**Reply by dpetka2001** [reply]
Replace the previous with `{ "<leader>w", desc = "Save file" }`

**Reply by blancpain** [reply]
Great - thanks a bunch - I was trying to set this under 'keys' rather than 'spec'. Works now.
